pkg/commands/game/add_user.go

```diff
@@ -0,0 +1,112 @@
+package game
+
+import (
+	"fmt"
+
+	"github.com/itfactory-tm/thomas-bot/pkg/command"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/itfactory-tm/thomas-bot/pkg/sudo"
+)
+
+// adduser contains the bob!adduser and bob!remuser command
+type AddUserCommand struct{}
+
+// NewAddUserCommand gives a new AddUserCommand
+func NewAddUserCommand() *AddUserCommand {
+	return &AddUserCommand{}
+}
+
+// Register registers the handlers
+func (m *AddUserCommand) Register(registry command.Registry, server command.Server) {
+	registry.RegisterMessageCreateHandler("adduser", m.addUser)
+	registry.RegisterMessageCreateHandler("remuser", m.remUser)
+}
+
+func (m *AddUserCommand) addUser(s *discordgo.Session, msg *discordgo.MessageCreate) {
+	if !(sudo.IsItfAdmin(msg.Author.ID) || sudo.IsAdmin(msg.Author.ID)) {
+		s.ChannelMessageSend(msg.ChannelID, fmt.Sprintf("%s is not in the sudoers file. This incident will be reported.", msg.Author.ID))
+		return
+	}
+
+	matched := msg.Message.Mentions
+	if len(matched) < 1 {
```

Comment:
What if 2 users are specified? Maybe we should error in that case, or loop over them

Reply:
It selects the first mentionned user, but looping over multiple users sounds useful!

Reply:
I'll also change matched to mention as matched doesn't really make sense in this context